components/core/tools/scripts/utils/build.py

```
@@ -0,0 +1,117 @@
+import argparse
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)